app/experimenter/outcomes/__init__.py

```diff
@@ -0,0 +1,57 @@
+import os
+from dataclasses import dataclass
+
+import toml
+from django.conf import settings
+
+from experimenter.experiments.constants import NimbusConstants
+
+
+@dataclass
+class Outcome:
+    application: str
+    app_name: str
+    description: str
+    friendly_name: str
+    slug: str
+
+
+class Outcomes:
+    _outcomes = None
+
+    @classmethod
+    def _load_outcomes(cls):
+        outcomes = []
+
+        for app_name in os.listdir(settings.JETSTREAM_CONFIG_OUTCOMES_PATH):
+            app_path = os.path.join(settings.JETSTREAM_CONFIG_OUTCOMES_PATH, app_name)
+
+            for outcome_name in os.listdir(app_path):
+                outcome_path = os.path.join(app_path, outcome_name)
+
+                with open(outcome_path, "r") as outcome_file:
+                    outcome_toml = outcome_file.read()
```

Comment:
It looks like the jetstream-config repo has validation for these files, but would it be handy to have some kind of last-ditch exception handler here to log errors just in case parsing fails during experimenter startup? 

Probably small chance of it happening and we'll see a toml-related error in CI, so maybe not worth much fuss

Reply:
Yeah I think if they don't adhere to the schema this will just straight up break and complain about a key error which is all the info I think we'll need to debug.  And it'll break in CI.